crates/swift-bridge-ir/src/parse/parse_extern_mod.rs

```diff
@@ -180,7 +179,32 @@ impl<'a> ForeignModParser<'a> {
                                 .push(ParseError::ArgCopyAndRefMut { arg: arg.clone() });
                         }
                     }
-
+                    if let Some(ref args) = attributes.args_into {
+                        for arg in args.iter() {
+                            let mut count = 0;
+                            for func_arg in func.sig.inputs.iter() {
+                                match func_arg {
+                                    FnArg::Typed(pat_ty) => {
+                                        let func_arg_name =
+                                            pat_ty.pat.to_token_stream().to_string();
+                                        let arg_name = arg.to_token_stream().to_string();
+                                        if func_arg_name != arg_name {
+                                            count += 1;
+                                        }
+                                    }
+                                    _ => {
+                                        todo!("Add a test that hits this case")
```

Comment:
This will panic if you use `args_into` on a method:

```rust
#[swift_bridge::bridge]
mod ffi {
    extern "Rust" {
        type SomeType;
        #[swift_bridge(args_into = (foo))]
        fn some_function(&self);
    }
}
```